tests/upstream-e2e/run-tests.sh

```diff
@@ -16,7 +16,7 @@ tar xzvf ${OUT_TAR} -C ${OUT_DIR}
 ./${OUT_DIR}/kubernetes/test/bin/e2e.test \
     -ginkgo.v \
     -ginkgo.focus='External.Storage' \
-    --ginkgo.skip='disruptive' \
+    --ginkgo.skip='\[Disruptive\]' \
```

Comment:
was this not correctly skipping the disruptive test before?

Reply:
When I enabled xfs and started testing locally, I could see two disruptive tests running and failing, not sure why they were not getting skipped with previous setting. Hence, I updated it to match with the string I was getting in logs than using the name defined in testsuite (https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/testsuites/disruptive.go#L45)